README.md

```diff
@@ -1,14 +1,138 @@
-# ISO 8583 Financial Messaging for Ruby
+### ISO 8583 Financial Messaging for Ruby
```

Comment:
Also corrected in the cleanup branch...

    https://github.com/a2800276/8583/tree/MaG21-rspec-tests